test/testSeries.js

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (C) 2021  Akash Gupta
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import bookbrainzData from './bookshelf';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import faker from 'faker';
+import {truncateTables} from '../lib/util';
+
+
+chai.use(chaiAsPromised);
+const {expect} = chai;
+const {
+	AliasSet, Annotation, Disambiguation, Editor, EditorType, Entity, Gender,
+	IdentifierSet, RelationshipSet, Revision, Series, SeriesOrderingType, UserCollection, UserCollectionItem, bookshelf
+} = bookbrainzData;
+
+const genderData = {
+	id: 1,
+	name: 'test'
+};
+const editorTypeData = {
+	id: 1,
+	label: 'test_type'
+};
+const editorData = {
+	genderId: 1,
+	id: 1,
+	name: 'bob',
+	typeId: 1
+};
+const orderTypeData = {
+	id: 1,
+	label: 'Manual'
+};
+
+const setData = {id: 1};
+
+const aBBID = faker.random.uuid();
+const aBBID2 = faker.random.uuid();
+
+const seriesAttribs = {
+	aliasSetId: 1,
+	bbid: aBBID,
+	entityType: 'Author',
+	identifierSetId: 1,
+	orderingId: 1,
```

Comment:
I think orderingTypeID would be more fitting, which would mean renaming the sql table to ordering_type_id